lib/quelink-mg/at/base.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class InvalidATGTBSIException < StandardError; end
+class InvalidATGTException < StandardError; end
```

Comment:
Raczej myślałem, by każda klasa miała jednak osobny błąd. Nawet jak nie jest obecnie rozróżniane.

Ale to można póżniej też dodać